src/components/FormElements/FormikRadioGroup/FormikRadioGroup.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from "react";
+
+import { FormControl } from "@material-ui/core";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import FormLabel from "@material-ui/core/FormLabel";
+import Radio from "@material-ui/core/Radio";
+import { RadioGroupProps } from "@material-ui/core/RadioGroup";
+import { Field, useFormikContext } from "formik";
+import { RadioGroup } from "formik-material-ui";
+
+type RadioValue = string | number;
+
+type RadioOption =
+  | {
+      label: React.ReactNode;
+      value: RadioValue;
+    }
+  | RadioValue;
```

Comment:
instead of having multiple options, it would be more simple to force it to always be a `label` and `value`, having these two different options makes it more complicated to maintain and provides very little benefit
```jsx
type RadioOption = {
  label: React.ReactNode;
  value: RadioValue;
}
```